src/Api/Auth/Models/Request/AdminAuthRequestUpdateRequestModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Bit.Api.Auth.Models.Request;
+
+public class AdminAuthRequestUpdateRequestModel
+{
+    public string Key { get; set; }
```

Comment:
`Key` --> `EncrypedUserKey` to match client changes?

Reply:
Ah, guess it would help to push my changes 🤦